storage_controller/migrations/2024-08-23-102952_safekeepers/up.sql

```diff
@@ -0,0 +1,17 @@
+-- very much a copy of cplane version, except that the projects_count is not included
+CREATE TABLE safekeepers (
+	-- the surrogate identifier defined by control plane database sequence
+	id BIGINT PRIMARY KEY,
+	created_at TIMESTAMPTZ NOT NULL DEFAULT now(),
+	updated_at TIMESTAMPTZ NOT NULL DEFAULT now(),
+	region_id TEXT NOT NULL,
+	version BIGINT NOT NULL,
```

Comment:
This is an interesting thought and might potentially be useful later.

Reply:
I think the AZ will be most useful initially, but who knows.